src/DependencyInjection/Configuration.php

```diff
@@ -99,6 +100,12 @@ public function getConfigTreeBuilder()
                     ->end()
                 ->end()
 
+                ->scalarNode('default_controller')
```

Comment:
Maybe we could also check here if the class is a valid controller.

Reply:
How would you do this? We don't have an interface for the CRUDControllers

Reply:
Maybe we must introduce one with this extension point, ensuring required methods are available. Since it wasn't configurable before, I think it could be done respecting BC.